### PR TITLE
Dw 4295 add email template to user pool

### PR DESCRIPTION
**Pull request by MattToczek**
Email template being fetched from storage S3 and added to cognito userpool configuration, to be used in the onboarding email and create continuity between the emails that’ll  we send to users.

### CODE REVIEW

File: terraform/deploy/cognito/variables.tf [hunk 1]
```diff
@@ -18,3 +18,7 @@ variable "name_prefix" {
   description = "(Optional) Name prefix for resources we create, defaults to repository name"
   default     = "analytical-env-cognito"
 }
+
+variable "onboarding_email_file_path" {
+  description = "File path to onboarding email template"
```
Comment (dan-hill2802) [inline terraform/deploy/cognito/variables.tf:23]:
```suggestion
  description = "(Required) File path to onboarding email template"
```

File: ci/aws-analytical-env/jobs/cognito_mgmt-dev.yml [hunk 2]
```diff
@@ -8,6 +8,8 @@ jobs:
         trigger: true
       - get: custom-auth-lambda-release
         trigger: true
+      - get: onboarding-email-template
+          trigger: true
```
Comment (dan-hill2802) [inline ci/aws-analytical-env/jobs/cognito_mgmt-dev.yml:12]: indentation is wrong

File: ci/aws-analytical-env/jobs/pr.yml [hunk 3]
```diff
@@ -63,6 +63,8 @@ jobs:
       - get: custom-auth-lambda-release
         version: { tag: '0.0.20' }
         trigger: true
+      - get: onboarding-email-template
+        trigger: true
```
Comment (dan-hill2802) [inline ci/aws-analytical-env/jobs/pr.yml:67]: should be `false` for PR job

Comment (MattToczek) [reply ci/aws-analytical-env/jobs/pr.yml:67]: I thought that but, the one above (custom-auth-lambda-release) is set to true so I thought I must be wrong. I've changed it now.

**Pull request merged**